packages/template-retail-react-app/app/static/translations/compiled/en-US.json

```diff
@@ -2275,6 +2275,18 @@
       "value": "Starting at"
     }
   ],
+  "product_view.button.add_bundle_to_cart": [
```

Comment:
You may want to copy this over en-GB too

Reply:
Added to `translations/en-GB.json` and then ran `npm run build-translations`